src/services/drive/add-file.ts

```diff
@@ -389,7 +391,7 @@ export default async function(
 	file.userId = user ? user.id : null;
 	file.userHost = user ? user.host : null;
 	file.folderId = folder !== null ? folder.id : null;
-	file.comment = comment;
+	file.comment = truncate(comment, DB_MAX_IMAGE_COMMENT_LENGTH);
```

Comment:
This is also used outside of ActivityPub, so it may be better to limit `image.name` below. 
https://github.com/misskey-dev/misskey/blob/cc756209764bef11709c71352b1402514174162a/src/remote/activitypub/models/image.ts#L31

Reply:
I think the current way is better, see the commit message:
> Truncating could also be done in `src/remote/activitypub/models/image.ts` or `src/services/drive/upload-from-url.ts` but both of those will use the code in `src/services/drive/add-file.ts` in the end. So this seems like the safest approach.

Reply:
`src/services/drive/add-file.ts` also works for local uploads.
I don't think truncate will occur for local uploads.
I think it is unsafe to ignore unexpected patterns and continue.

Reply:
I see, local changes will now definitely cause a hard error if trying to insert into the database. This is a double check from the validator.